src/tests/libecalc/input/validation/snapshots/test_validation_json_schemas/test_json_schema_changed/schemas.json

```diff
@@ -1494,55 +1506,70 @@
                     "title": "YamlTrain[YamlCompressor]",
                     "type": "object"
                 },
-                "YamlVentingEmitter": {
+                "YamlVentingEmission": {
                     "additionalProperties": false,
                     "properties": {
-                        "CATEGORY": {
-                            "anyOf": [
+                        "NAME": {
+                            "description": "Name of emission",
+                            "title": "NAME",
+                            "type": "string"
+                        },
+                        "RATE": {
+                            "allOf": [
                                 {
-                                    "$ref": "#/definitions/ConsumerUserDefinedCategoryType"
-                                },
+                                    "$ref": "#/definitions/YamlRate"
+                                }
+                            ],
+                            "description": "The emission rate",
+                            "title": "RATE"
+                        }
+                    },
+                    "required": [
+                        "NAME",
+                        "RATE"
+                    ],
+                    "title": "YamlVentingEmission",
+                    "type": "object"
+                },
+                "YamlVentingEmitter": {
+                    "additionalProperties": false,
+                    "properties": {
+                        "COMPONENT_TYPE": {
```

Comment:
This is not something the user should provide, right? If we are going to have a type in yaml, it should be called `TYPE`?

Reply:
The user should not provide this. Compoent type is just set internally.